Yeti-experience/draft-song-yeti-testbed-experience.txt

```diff
@@ -277,134 +319,118 @@ Internet-Draft      Yeti DNS Testbed: Interim Results          June 2017
 
 
-   Figure 1.  The topology of Yeti testbed
 
-3.1.  Distribution Master
 
-   As shown in figure 1, the Yeti Root system takes the IANA root zone
-   and performs minimal changes needed to serve the zone from the Yeti
-   root servers instead of the IANA root servers.  In Yeti, this
-   modified root zone is generated by the Yeti Distribution Masters
-   (DM), which provide it to the Yeti root servers.
 
-   The zone generation process is:
 
-   o  DM downloads the latest IANA root zone at a certain time
 
-   o  DM makes modifications to change from the IANA to Yeti root
-      servers
 
-   o  DM signs the new Yeti root zone with Yeti key
 
-   o  DM publishes the new Yeti root zone to Yeti root servers
 
 
 
+Song, et al.            Expires December 18, 2017               [Page 6]
+
+Internet-Draft      Yeti DNS Testbed: Interim Results          June 2017
 
 
-Song, et al.            Expires December 11, 2017               [Page 6]
+                               ,----------------.
+                          ,-- / IANA Root Zone / ---.
+                          |  `----------------'     |
+                          |            |            |
+                          |            |            |          Root Zone
+  ,--------------.    ,---V---.    ,---V---.    ,---V---.
+  | Yeti Traffic |    | BII   |    | WIDE  |    | TISF  |
+  | Collection   |    |  DM   |    |  DM   |    |  DM   |
+  `----+----+----'    `---+---'    `---+---'    `---+---'
+       |    |       ,-----'    ,-------'            `----.
+       |    |       |          |                         |
+       ^    ^       |          |                         |     Yeti-Root
+       |    |   ,---V---.  ,---V---.                 ,---V---.
+       |    `---+ Yeti  |  | Yeti  |  . . . . . . .  | Yeti  |
+       |        | Root  |  | Root  |                 | Root  |
+       |        `---+---'  `---+---'                 `---+---'
+       |            |          |                         |
+       |            |          |                         |     DNS Response
+       |          ,--V----------V-------------------------V--.
+       `----------+                Yeti Resolvers            |
+                  `-----------------------+------------------'
+                                          |
+                                          |                    DNS Response
+                  ,-----------------------V------------------.
+                  |              Yeti Stub Resolvers         |
+                  `------------------------------------------'
+
+
+                        Figure 2: Testbed Data Flow
+
+3.1.  Root Zone Retrieval
+
+   Since Yeti DNS servers cannot receive DNS NOTIFY [RFC1996] messages
+   from the Root Server System, a polling approach is used.  Each Yeti
+   Distribution Master (DM) requests the root zone SOA record from
+   XXX(which servers?)XXX and performs a zone transfer if the SOA.SERIAL
```

Comment:
The root zone is currently downloaded using AXFR from F.ROOT-SERVERS.NET:
    $ dig -t axfr . @f.root-servers.net.

more information of DM setup : https://github.com/BII-Lab/Yeti-Project/blob/master/doc/Yeti-DM-Setup.md